src/modules/posts/store/Reducer.ts

```diff
@@ -13,27 +13,43 @@ export interface Post {
 
 export interface PostsState {
   items: Post[];
+  errors: {
+    [key: string]: string
+  }
 }
 
 const initialState: PostsState = {
   items: [],
+  errors: {}
 };
 
 export class PostsReducer extends ImmerReducer<PostsState> {
   // eslint-disable-next-line
   fetchPostsRequest(): void {}
 
+  // eslint-disable-next-line
```

Comment:
Why the need for `eslint-disable-next-line`?

Reply:
That is a very good question